src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -206,6 +206,21 @@ impl CrashReport {
             } else {
                 Some(crash_log.minimized_stack_function_names_sha256(minimized_stack_depth))
             };
+
+        let minimized_stack_function_lines = if crash_log.minimized_stack_function_lines.is_empty()
+        {
```

Comment:
any { } layout style  preference there ?

Reply:
It follows default `cargo fmt`.